packages/components/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import * as RadixCheckbox from "@radix-ui/react-checkbox";
+import { CheckIcon } from "@radix-ui/react-icons";
+import { styled } from "../../stitches.config";
+
+const StyledBox = styled(RadixCheckbox.Root, {
+  backgroundColor: "transparent",
+  borderWidth: 1,
+  borderColor: "transparent",
+  padding: 0,
+  width: 24,
+  height: 24,
+  display: "inline-flex",
+  alignItems: "center",
+  justifyContent: "center",
+  defaultVariants: {
+    checkDisplay: "unchecked",
+  },
+  variants: {
+    disabledClick: {
+      true: {
+        backgroundColor: "#FFEED9",
```

Comment:
we use this in a few places, we likely will want to pull this to a theme variable, probably separated.

Reply:
yea we should add it to here, could be a separate PR

https://github.com/CabinDAO/Topo/blob/main/packages/theme/src/index.ts#L23-L34